.github/workflows/build-coq-bignums.yml

```
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '**'
+
```

BTW @Zimmi48 I've just [disabled the Travis CI cron build](https://travis-ci.com/github/coq/bignums/settings).
So @proux01 you might want to re-add some daily build in the workflow:

```suggestion
  schedule:
  # test master every day at 05:00 UTC
    - cron: '0 5 * * *'

```